### DISCUSSION TITLE
Show time remaining with loading toast?

### DISCUSSION THREAD

**Discussion by vedantroy** [fkhadra/react-toastify#768] [Q&A]
Is there a way to use `toast.loading` while also having the bar at the bottom that shows how much time is left in the toast?

**Comment by fkhadra** [comment]
Hey @vedantroy, yes it's possible both with the controlled toast or if you already know the time you want to display the notification.

```jsx
import {Icons, toast } from "react-toastify"

//10sec
const taskDuration = 10000

toast("Doing something...", {
  icon: Icons.spinner,
  autoClose: taskDuration
})

```
